deploy/src/erc_to_native/home.js

```diff
@@ -123,31 +149,63 @@ async function deployHome() {
   assert.strictEqual(Web3Utils.hexToNumber(txUpgradeToHomeBridge.status), 1, 'Transaction Failed')
   homeNonce++
 
-  console.log('\ninitializing Home Bridge with following parameters:\n')
-  console.log(`Home Validators: ${storageValidatorsHome.options.address},
+  let initializeHomeBridgeData
+  homeBridgeImplementation.options.address = homeBridgeStorage.options.address
+
+  if (useRewardableValidators) {
+    console.log('\ndeploying implementation for fee manager')
+    const feeManager = await deployContract(FeeManagerErcToNative, [], {
+      from: DEPLOYMENT_ACCOUNT_ADDRESS,
+      nonce: homeNonce
+    })
+    console.log('[Home] feeManager Implementation: ', feeManager.options.address)
+    homeNonce++
+
+    const feeInWei = Web3Utils.toWei(BRIDGE_FEE.toString(), 'ether')
+    console.log('\ninitializing Home Bridge with fee contract:\n')
+    initializeHomeBridgeData = await homeBridgeImplementation.methods
+      .rewardableInitialize(
+        storageValidatorsHome.options.address,
+        HOME_DAILY_LIMIT,
+        HOME_MAX_AMOUNT_PER_TX,
+        HOME_MIN_AMOUNT_PER_TX,
+        HOME_GAS_PRICE,
+        HOME_REQUIRED_BLOCK_CONFIRMATIONS,
+        BLOCK_REWARD_ADDRESS,
+        FOREIGN_DAILY_LIMIT,
+        FOREIGN_MAX_AMOUNT_PER_TX,
+        HOME_BRIDGE_OWNER,
+        feeManager.options.address,
+        feeInWei
+      )
+      .encodeABI({ from: DEPLOYMENT_ACCOUNT_ADDRESS })
+  } else {
+    console.log('\ninitializing Home Bridge with following parameters:\n')
+    console.log(`Home Validators: ${storageValidatorsHome.options.address},
```

Comment:
Seems that these lines will not be reported for the case with the rewardable bridge

Reply:
Fixed here https://github.com/poanetwork/poa-bridge-contracts/pull/134/commits/e56b51d669abd2465d51ab56b7269b43c774a4b3
Also added some variables that were missing on the log